sos/report/plugins/hbn.py

```diff
@@ -12,13 +12,28 @@
 
 class HBN(Plugin, IndependentPlugin):
 
-    short_desc = 'HBN (Host Based Network)'
+    short_desc = 'HBN (Host Based Networking)'
     plugin_name = "hbn"
+    vsctl = "ovs-vsctl"
     packages = ('hbn-repo',)
 
     def setup(self):
         self.add_copy_spec([
             '/etc/mellanox',
             '/var/log/doca/hbn',
             '/var/lib/hbn/etc/cumulus',
+            '/var/log/containers',
+            '/var/log/sfc-install.log',
+            '/tmp/sf_devices',
+            '/tmp/sfr_devices',
+            '/tmp/sfc-activated',
+        ])
+
+        self.add_journal(units="sfc")
+        self.add_journal(units="sfc-state-propagation")
+
+        self.add_cmd_output([
+            "crictl ps -a",
+            "mlnx-sf -a show",
+            "mlnx-sf -a show --json --pretty",
```

Comment:
why do we need this command twice, can't we use the second one?

should this be part of a different plugin that only tracks SFs?

Reply:
The output of those two are slightly different.
The first one is a shorter CLI like output while the second one is a more verbose json version with more info.
I believed for troubleshooting, it was useful to first look at the shorter ones output  to see if all the expected SFs are there and use the second one only for more detailed troubleshooting if we suspect the issue is with SF creation.

Reply:
will do